### PR TITLE
Fix varedited areas on Meta.

### PR DESCRIPTION
**Pull request by warriorstar-orion**
## What Does This PR Do
Removes varedited areas on Meta. Buncha changes here so I'm listing them out:

1. `/area/quartermaster/miningdock` is no longer referred to as `\improper Mining Office`.
2. The area outside cargo, previously typed as `/area/construction/hallway` (???) is now a new type, `/area/quartermaster/lobby`.
3. The garden, previously typed as `/area/hallway/secondary/construction` (?!?!) now uses the existing `/area/hallway/secondary/garden`.
4. The vacant food service area outside cargo, previously an `/area/crew_quarters/cafeteria` with a name of `\improper Cafe` is a new type, `/area/crew_quarters/cafe`, with the unused former type removed, because yes I am pedantic enough to distinguish between a cafe and a cafeteria and no one can stop me. Also turns out the chef could have used CQC in there prior to this. Oh well.
5. The `/area/crew_quarters/fitness` area renamed `\improper Recreation Area` is now `/area/crew_quarters/recreation`.
6. The `/area/hallway/secondary/entry` containing arrivals has lost its custom name. I don't see why this specific type should be named "Arrivals" solely on one of our maps so I just removed the custom name instead of creating a new area type for it.
7. Similarly for `/area/hallway/secondary/exit`.
8. `/area/crew_quarters/locker/locker_toilet` now uses the same name as other maps.
9. The Captain's areas, previously `/area/crew_quarters/captain` with the name "Captain's Quarters", is now split into the pre-existing `/area/crew_quarters/captain` and `/area/crew_quarters/captain/bedroom`.
10. The "bridge hallway", previously a custom `/area/bridge/meeting_room`, is now a new type `/area/bridge/hall`.
11. The AI sat has been standardized to use the areas introduced in #20120.
12. The auxiliary public toilets have been moved from a varedited `/area/crew_quarters/toilet` to a new type, `/area/crew_quarters/toilet/aux`.
13. The "Shared Engineering Storage" is now just the default `/area/engine/equipmentstorage`.
14. The Toxins Launch Room had an unnecessary rename removed.

Part of #20018, fixes for which I'm splitting into separate PRs because I'm not going to attempt to manage one PR that touches five different maps if I don't absolutely have to.

## Why It's Good For The Game
Varedited areas bad.

## Testing
Started server on Meta, joined, observed, VVed object locs. Checked no runtimes were logged. I didn't VV every affected area because there were just too many changes for this map.

No visual changes.

### CODE REVIEW

File: code/game/area/ss13_areas.dm [hunk 1]
```diff
@@ -893,6 +896,11 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	icon_state = "toilet"
 	sound_environment = SOUND_AREA_SMALL_ENCLOSED
 
+/area/crew_quarters/toilet/aux
+	name = "\improper Auxiliary Toilets"
+	icon_state = "toilet"
```
Comment (S34NW) [inline code/game/area/ss13_areas.dm:901]: inherited var

File: code/game/area/ss13_areas.dm [hunk 2]
```diff
@@ -893,6 +896,11 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
...
+	sound_environment = SOUND_AREA_SMALL_ENCLOSED
```
Comment (S34NW) [inline code/game/area/ss13_areas.dm:902]: and here

File: code/modules/martial_arts/cqc.dm [hunk 3]
```diff
@@ -5,7 +5,6 @@
 	var/restraining = FALSE //used in cqc's disarm_act to check if the disarmed is being restrained and so whether they should be put in a chokehold or not
 	var/chokehold_active = FALSE //Then uses this to determine if the restrain actually goes anywhere
 	var/static/list/areas_under_siege = typecacheof(list(/area/crew_quarters/kitchen,
-														/area/crew_quarters/cafeteria,
```
Comment (S34NW) [inline code/modules/martial_arts/cqc.dm:8]: Funny maints chef gone? sad :(

**Review by S34NW — CHANGES_REQUESTED** [review]

File: code/game/area/ss13_areas.dm (hunk 1 above)
Comment (warriorstar-orion) [reply code/game/area/ss13_areas.dm]: Done

File: code/game/area/ss13_areas.dm (hunk 2 above)
Comment (warriorstar-orion) [reply code/game/area/ss13_areas.dm]: Done

File: code/modules/martial_arts/cqc.dm (hunk 3 above)
Comment (warriorstar-orion) [reply code/modules/martial_arts/cqc.dm:8]: There was a place in maints where chefs had CQC?

Comment (S34NW) [reply code/modules/martial_arts/cqc.dm:8]: `The vacant food service area outside cargo, previously an /area/crew_quarters/cafeteria`

**Review by S34NW — APPROVED** [review]

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**